tests/baselines/reference/baseTypeWrappingInstantiationChain.errors.txt

```diff
@@ -0,0 +1,35 @@
+tests/cases/compiler/baseTypeWrappingInstantiationChain.ts(1,21): error TS2686: A class must be declared after its base class.
+tests/cases/compiler/baseTypeWrappingInstantiationChain.ts(12,25): error TS2686: A class must be declared after its base class.
+
+
+==== tests/cases/compiler/baseTypeWrappingInstantiationChain.ts (2 errors) ====
+    class C<T1> extends CBase<T1> {

```

Comment:
Can you reorder these classes to fix the error? I'd rather keep types and symbols around for tests that are not meant to exercise the new error.